TombEngine/Scripting/Internal/ReservedScriptNames.h

```diff
@@ -109,6 +109,7 @@ static constexpr char ScriptReserved_EndLevel[]					= "EndLevel";
 static constexpr char ScriptReserved_GetSecretCount[]			= "GetSecretCount";
 static constexpr char ScriptReserved_SetSecretCount[]			= "SetSecretCount";
 static constexpr char ScriptReserved_AddSecret[]				= "AddSecret";
+static constexpr char ScriptReserved_SetFlyCheatEnabled[]		= "SetFlyCheatEnabled";
```

Comment:
Why not simply name it `EnableFlyCheat`? :)

Reply:
Had change it to EnableFlyCheat option. Builders can set in gameflow.lua: Flow.EnableFlyCheat(false)